release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -682,6 +710,22 @@ $ operator-sdk new <operator_name> --type=helm \
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1874754[*BZ#1874754*])
 
+* In {product-title} 4.6, the resource metrics API server provides support for custom metrics. The resource metrics API server does not implement the OpenAPI specification and the following messages are sent to the `kube-apiserver` logs every 60 seconds:
++
+[source,terminal]
+----
+controller.go:114] loading OpenAPI spec for "v1beta1.metrics.k8s.io" failed with: OpenAPI spec does not exist
+controller.go:127] OpenAPI AggregationController: action for item v1beta1.metrics.k8s.io: Rate Limited Requeue.
+----
++
+In some cases, the above errors might cause the `KubeAPIErrorsHigh` alert to fire.
```

Comment:
```suggestion
In some cases, these errors might cause the `KubeAPIErrorsHigh` alert to fire, but the underlying issue does not degrade {product-title} functionality.
```

> 🚫  above adj
Do not use to indicate a relative location in a document, as in “the above restrictions.” Use “previous” or “preceding.”

Reply:
Thank you. I have now combined the two sentences. I have reworded the second one slightly, because I think it is important to not to state absolutely that this does not degrade functionality:

"In some cases, these errors might cause the `KubeAPIErrorsHigh` alert to fire, but the underlying issue is not known to degrade {product-title} functionality."